protocol-test-util/pom.xml

```diff
@@ -85,6 +85,37 @@
       <artifactId>zeebe-atomix-cluster</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>io.camunda</groupId>
+      <artifactId>zeebe-protocol-util</artifactId>
+    </dependency>
+
+    <!-- required for the protocol factory -->
+    <dependency>
+      <groupId>org.jeasy</groupId>
+      <artifactId>easy-random-core</artifactId>
+      <version>5.0.0</version>
```

Comment:
🔧 Please define the dependency in the parent's dependencyManagement section, and remove the version declaration here.

Reply:
I did, I just forgot to remove the version declaration :see_no_evil: